Unix Time.txt

```diff
@@ -0,0 +1,50 @@
+OBJECT Codeunit 50100 Unix Time
+{
+  OBJECT-PROPERTIES
+  {
+    Date=06-09-17;
+    Time=06:32:28;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+            MESSAGE(FORMAT(GetUnixTime(CURRENTDATETIME)));
+          END;
+
+  }
+  CODE
+  {
+
+    PROCEDURE GetUnixTime@6150645(FromDateTime@6150614 : DateTime) UnixTimeStamp : Integer;

```

Comment:
Did you consider adding the function to codeunit 10 instead  - and write it using a similar way as the function EvaluateUnixTimestamp(Timestamp : BigInteger) : DateTime ?